pallets/messages/src/tests.rs

```diff
@@ -238,226 +243,34 @@ fn on_initialize_should_add_messages_into_storage_and_clean_temp() {
 	});
 }
 
-#[test]
-fn get_messages_by_schema_with_valid_request_should_return_paginated() {
-	new_test_ext().execute_with(|| {
-		// arrange
-		let schema_id: SchemaId = 1;
-		let page_size = 3;
-		let from_index = 2;
-		let messages_per_block = vec![10, 0, 5, 2, 0, 3, 9];
-		populate_messages(schema_id, messages_per_block.clone(), PayloadLocation::OnChain);
-		let request = BlockPaginationRequest {
-			page_size,
-			from_block: 0,
-			to_block: messages_per_block.len() as u64,
-			from_index,
-		};
-
-		// act
-		let res = MessagesPallet::get_messages_by_schema(schema_id, request);
-
-		// assert
-		assert_ok!(&res);
-
-		let pagination_response = res.ok().unwrap();
-		assert_eq!(pagination_response.content.len() as u32, page_size);
-		assert_eq!(pagination_response.has_next, true);
-		assert_eq!(pagination_response.next_block, Some(0));
-		assert_eq!(pagination_response.next_index, Some(from_index + page_size));
-
-		let payload = Vec::from("{'fromId': 123, 'content': '232323114432'}".as_bytes());
-		assert_eq!(
-			pagination_response.content[0],
-			MessageResponse {
-				msa_id: Some(10),
-				payload: Some(payload.clone()),
-				index: from_index as u16,
-				provider_msa_id: 1,
-				block_number: 0,
-				payload_length: None,
-				cid: None
-			}
-		);
-	});
-}
-
-#[test]
-fn get_messages_by_schema_with_valid_request_should_return_next_block_and_index() {
```

Comment:
Moved to the Api test, and to the check_end_condition_and_set_next_pagination tests